README.md

```diff
@@ -94,7 +94,7 @@ This file will have to hold the following information:
 
 Optionally, the following information can be added:
 
-- `NEXT_PUBLIC_POSTHOG_KEY`: the name of the [PostgreSQL](https://www.postgresql.org/) database that will be created.
+- `NEXT_PUBLIC_POSTHOG_KEY`: the key of the [PostHog](https://posthog.com/) platform that will be created.
```

Comment:
Would you mind changing this to "the project key of your [PostHog](https://posthog.com/) service, for product analytics"? :pray:

Reply:
Sure, I already make some changes on this [commit](https://github.com/fahreziadh/platform/commit/bbe65a1c526bc8862f51daa161476a698c04f450)